cdap-ui/app/hydrator/controllers/create/create-studio-ctrl.js

```diff
@@ -77,6 +77,9 @@ class HydratorPlusPlusStudioCtrl {
       let config = {};
       config.artifact = artifact;
       HydratorPlusPlusConfigActions.initializeConfigStore(config);
+      if (rConfig.upgrade){
```

Comment:
nit: Could we have a blank line like this? `if (rConfig.upgrade) {`

Reply:
Fixed.